modules/ROOT/pages/ch2-api.adoc

```diff
@@ -0,0 +1,376 @@
+
+*CH 1.0 example*
+
+----
+curl -X GET \
+  --url https://anypoint.mulesoft.com/cloudhub/api/applications \
+  -H 'authorization: Bearer AUTH_BEARER_TOKEN' \
+  -H 'X-ANYPNT-ENV-ID: ENV_ID' \
+  -H 'X-ANYPNT-ORG-ID: ORG_ID'
+----
+
+
+== Data Format
+
+*CH 1 Content* 
+
+*DO THESE APPLY TO CH2?*
+
+Resources and methods that return or accept a type use the JSON data format.
+
+Here is an example of data received in JSON format in response to a request to get an application:
+
+[source,json,linenums]
+----
+{
+  "versionId": "xxxxxxxx",
+  "domain": "hello",
+  "fullDomain": "hello.us-e1.cloudhub.io",
+  "properties": {
+    "foo":"bar"
+  },
+  "propertiesOptions": {},
+  "status": "STARTED",
+  "workers": {
+    "type": {
+      "name": "Micro",
+      "weight": 0.1,
+      "cpu": "0.1 vCores",
+      "memory": "500 MB memory"
+    },
+    "amount": 1,
+    "remainingOrgWorkers": 223.7,
+    "totalOrgWorkers": 1000.0
+  },
+  "workerStatuses": [
+    {
+      "id": "i-xxxxxxxx",
+      "host": "xxx.xxx.xxx.xxx",
+      "port": 0,
+      "status": "STARTED",
+      "deployedRegion": "us-east-1",
+      "staticIPEnabled": false
+    }
+  ],
+  "lastUpdateTime": 1589868960908,
+  "fileName": "hello.zip",
+  "muleVersion": {
+    "version": "4.3.0",
+    "updateId": "xxxxxxxx",
+    "latestUpdateId": "xxxxxxxx",
+    "endOfSupportDate": 1633737600000
+  },
+  ...
+}
+----
+
+== Rate Limits
+
+*CH 1 Content* 
+
+*DO THESE APPLY TO CH2?*
+
+The CloudHub 2.0 management API enforces global usage limits, with some exceptions.
+
+
+[[global-rate-limits]]
+=== Global Rate Limits for the CloudHub 2.0 Management API
+
+The following table lists the rate limits for the CloudHub 2.0 management API endpoints except those listed in <<global-rate-limit-exceptions>>.
+
+When the app reaches the rate limit, CloudHub 2.0 returns a 503 status code.
+
+[%header,cols="20,20,20,10"]
+|===
+|Endpoints |Control Plane | Rate Limit Per Client IP Address | Status Code
+.3+|All except specified endpoints|U.S.|75 requests per second  .3+| `503`
+|EU .2+| 25 requests per second
+|Government Cloud
+|===
+
+[[global-rate-limit-exceptions]]
+=== Exceptions to the Global Rate Limits
+
+The following table lists the exceptions to the <<global-rate-limits>>.
+
+When the app reaches the rate limit, CloudHub 2.0 returns the status codes shown here.
+
+[%header,cols="20,20,20,10"]
+|===
+|Endpoints|Control Plane | Rate Limit Per Client IP Address| Status Code
+4+|*Endpoints under* `/api`
+.3+|`/notifications`|U.S.|75 requests per minute .3+| `503`
+|EU .2+|25 requests per minute
+|Government Cloud
+4+|*Endpoints under* `*v2/applications/{domain}*`
+|`/logs` .4+|All .3+|10 requests per second  .3+| `429`
+|`/deployments/{deploymentId}/logs`
+|`/instances/{instanceId}/logs`
+|`/instances/{instanceId}/log-file`|1 request per minute  | `500`
+|===
+
+
+
+== Status Codes and Error Handling
+
+*CH 1 Content* 
+
+*DO THESE APPLY TO CH2?*
```

Comment:
Yes.

Reply:
All of the CH1 content? (Data Format through Status Codes)

Reply:
Yes to all: data format, rate limits, and status codes.

Reply:
Thanks!